qdrant/manifest.json

```diff
@@ -0,0 +1,74 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bee8e975-219c-4673-83c0-9fa68b973298",
+  "app_id": "qdrant",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "A high-performance vector search engine/database.",
+    "title": "Qdrant",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "A Qdrant Monitoring Dashboard for Datadog",
+        "image_url": "images/qdrant_dash.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "REST metrics from the Qdrant Datadog dashboard",
+        "image_url": "images/qdrant_rest.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "cluster metrics from the Qdrant Datadog dashboard",
+        "image_url": "images/qdrant_cluster.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "gRPC metrics from the Qdrant Datadog dashboard",
+        "image_url": "images/qdrant_grpc.png"
```

Comment:
The images are great but it is super helpful for customers to see data in the dashboard. This givezs an accurate representation on what to expect. Can you make sure that the dashboard is filled with data (can be dummy data)? In this case it looks like it's only showing at the end of the graph and a most images show 0 responses! For context, this is going to show as the images on your tile so it will be the first thing users will see before they read the overview/click install!

Reply:
Hey. Thanks for the info. I'll connect to a production Qdrant instance to get the metrics.

Reply:
[`72bc153` (#2422)](https://github.com/DataDog/integrations-extras/pull/2422/commits/72bc1539527f0a6870c00a36470de2bbe25dba85)